identity-account/src/stronghold/context.rs

```diff
@@ -78,49 +65,57 @@ async fn clear_expired_passwords() -> Result<()> {
   Ok(())
 }
 
-impl Context {
-  pub(crate) fn get() -> Result<&'static Self> {
-    static __THIS: OnceCell<Context> = OnceCell::new();
-    static __POOL: OnceCell<AsyncRuntime> = OnceCell::new();
-    static __SWEEP: Once = Once::new();
+static CONTEXT: Lazy<core::result::Result<Context, String>> = Lazy::new(|| {
+  let (sender, receiver) = std::sync::mpsc::channel();
 
-    let this: &'static Self = __THIS.get_or_try_init::<_, Error>(|| {
-      let system: ActorSystem = SystemBuilder::new()
-        // Disable the default actor system logger
-        .log(slog::Logger::root(slog::Discard, slog::o!()))
-        .create()?;
+  thread::spawn(move || {
+    let system_runner = System::new();
 
-      let stronghold: Stronghold = Stronghold::init_stronghold_system(system, Vec::new(), Vec::new());
-      let database: Arc<AsyncMutex<Database>> = Arc::new(AsyncMutex::new(Database::new(stronghold)));
+    // TODO: convert error properly once anyhow is no longer used
+    let stronghold = system_runner
+      .block_on(Stronghold::init_stronghold_system(vec![], vec![]))
+      .map_err(|err| err.to_string());
 
-      Ok(Self {
+    let context = stronghold.map(|sh| {
+      let database: Arc<AsyncMutex<Database>> = Arc::new(AsyncMutex::new(Database::new(sh)));
+      Context {
         database,
         runtime: Runtime::new(),
```

Comment:
this should be fine, as stronghold implicitly runs on a single threaded tokio runtime. access to strongholds functionality is synchronized.

Reply:
Do you still think it makes sense to make the `Stronghold` type clonable and get rid of the mutex here? This can come at a later point, I think, but it would be good to remove the double synchronization. In any case, thanks for taking the time to review!

Reply:
`Stronghold` can be made `clone`able, since it only stores addresses. One issue i see here, is the synchronization between the `actix` system and the `runtime` used by `identity`. "Issue" in that sense that we need to create two `runtime`s. Maybe there is a way to lift this issue in the future.